Unity 5/EditorToolkit/Assets/Babylon/Scripts/ExporterWindow.cs

```diff
@@ -41,7 +41,7 @@ public class ExporterWindow : EditorWindow
         bool showPreview = false;
         bool showCompiler = false;
         int buildResult = 0;
-        string defaultProjectFolder = String.Empty;
+        public string defaultProjectFolder = String.Empty;

```

Comment:
I'm exposing this because it would be useful for tools that are using this script to set their own output path.